memestra/version.py

```diff
@@ -0,0 +1,3 @@
+__version__ = '0.0.1'
```

Comment:
Nice!

Reply:
Actually is 0.0.3 now right :p
I'll fix it!